Index: user-service/src/main/java/com/training/userservice/repository/UserRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.training.userservice.repository;\r\n\r\nimport com.training.userservice.entity.User;\r\nimport org.springframework.data.jpa.repository.JpaRepository;\r\n\r\nimport java.util.Optional;\r\nimport java.util.UUID;\r\n\r\npublic interface UserRepository extends JpaRepository<User, UUID> {\r\n\r\n    Optional<User> findByEmail(String email);\r\n\r\n    boolean existsByEmail(String email);\r\n}\r\n
===================================================================
diff --git a/user-service/src/main/java/com/training/userservice/repository/UserRepository.java b/user-service/src/main/java/com/training/userservice/repository/UserRepository.java
--- a/user-service/src/main/java/com/training/userservice/repository/UserRepository.java	(revision 89a4be19ce183c940d2fd4fb661742eba1e134e9)
+++ b/user-service/src/main/java/com/training/userservice/repository/UserRepository.java	(date 1780843310430)
@@ -11,4 +11,4 @@
     Optional<User> findByEmail(String email);
 
     boolean existsByEmail(String email);
-}
+}
\ No newline at end of file
